test/core/extensions/crypto_extension_test.cpp

```diff
@@ -373,7 +373,7 @@ TEST_F(CryptoExtensionTest, Twox256) {
  * @when call recovery public secp256k1 uncompressed key
  * @then resulting public key is correct
  */
-TEST_F(CryptoExtensionTest, Secp256k1RecoverUncompressed) {
+TEST_F(CryptoExtensionTest, DISABLED_Secp256k1RecoverUncompressed) {
```

Comment:
Why did it was disable? Enable back this test or remove if that needless, pls.

Reply:
it's a temporary measure

Reply:
Already re-enabled after fix.